solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -850,10 +766,782 @@ describe("RandomBeacon - Relay", () => {
+                        groupThreshold
+                      )
+
+                    await expect(
+                      randomBeacon.notifyFailedHeartbeat(
+                        {
+                          groupId,
+                          failedMembersIndices: subsequentFailedMembersIndices,
+                          signatures,
+                          // Remove the first signing member index
+                          signingMembersIndices: signingMembersIndices.slice(1),
+                        },
+                        0
+                      )
+                    ).to.be.revertedWith("Unexpected signatures count")
+                  })
+                }
+              )
+
+              context("when first signing member index is zero", () => {
+                it("should revert", async () => {
+                  const { signatures, signingMembersIndices } =
+                    await signHeartbeatFailureClaim(
+                      members,
+                      0,
+                      group.groupPubKey,
+                      subsequentFailedMembersIndices,
+                      groupThreshold
+                    )
+
+                  signingMembersIndices[0] = 0
+
+                  await expect(
+                    randomBeacon.notifyFailedHeartbeat(
+                      {
+                        groupId,
+                        failedMembersIndices: subsequentFailedMembersIndices,
+                        signatures,
+                        signingMembersIndices,
+                      },
+                      0
+                    )
+                  ).to.be.revertedWith("Corrupted members indices")
+                })
+              })
+
+              context(
+                "when last signing member index is bigger than group size",
+                () => {
+                  it("should revert", async () => {
+                    const { signatures, signingMembersIndices } =
+                      await signHeartbeatFailureClaim(
+                        members,
+                        0,
+                        group.groupPubKey,
+                        subsequentFailedMembersIndices,
+                        groupThreshold
+                      )
+
+                    signingMembersIndices[signingMembersIndices.length - 1] = 65
+
+                    await expect(
+                      randomBeacon.notifyFailedHeartbeat(
+                        {
+                          groupId,
+                          failedMembersIndices: subsequentFailedMembersIndices,
+                          signatures,
+                          signingMembersIndices,
+                        },
+                        0
+                      )
+                    ).to.be.revertedWith("Corrupted members indices")
+                  })
+                }
+              )
+
+              context(
+                "when signing members indices are not ordered in ascending order",
```

Comment:
`signingMembersIndices[10] = signingMembersIndices[11]` makes them duplicate

```
tmp = signingMembersIndices[10]
signingMembersIndices[10] = signingMembersIndices[11]
signingMembersIndices[11] = tmp
```

would make the not ordered in ascending order